cmd/verify.go

```diff
@@ -174,6 +178,7 @@ func NewVerifyCmd(config *viper.Viper) *cobra.Command {
 	cmd.Flags().StringSliceVarP(&verifyOpts.Values, "set", "s", []string{}, "overrides a configuration, e.g: dummy.ok=false")
 
 	cmd.Flags().StringSliceVarP(&verifyOpts.ValueFiles, "set-values", "f", nil, "specify application and check configuration values in a YAML file or a URL (can specify multiple)")
+	cmd.Flags().StringVarP(&openshiftVersionFlag, "openshift-version", "V", "", "version of OpenShift used in the cluster")
 
```

Comment:
The `https://github.com/Masterminds/semver` package is already an implicit dependency, and it can be used to validate the user input (see `semver.Version`).

Reply:
Done. f3ae774f1f1304b39bdc942c7ad2bc304ad3ddf3